### PR TITLE
[INLONG-5039][Manager] Support delete command, update command, and query execution log for managerctl

### LINKED ISSUE #5039: [Feature][Manager] Support query resource execution log for managerctl
### Description

Support query resource execution log.

### Use case

Support query resource execution log.

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Yizhou-Yang**
Fixes #5039 

### Modifications

changed the command line tools, should have no impact on the manager's behavior.

### Verifying this change

Added test cases in TestCommand.

### CODE REVIEW

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/DeleteCommand.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Parameters(commandDescription = "Delete group by group id")
+    private static class DeleteGroup extends AbstractCommandRunner {
+
+        @Parameter()
+        private java.util.List<String> params;
+
+        @Parameter(names = {"--group", "-g"}, required = true, description = "group id")
+        private String groupId;
```
Comment (haifxu) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/DeleteCommand.java:44]: `group id` -> `inlong group id`

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/UpdateCommand.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    @Parameters(commandDescription = "Update group by json file")
+    private static class UpdateGroup extends AbstractCommandRunner {
+
+        @Parameter()
+        private java.util.List<String> params;
+
+        @Parameter(names = {"--group", "-g"}, required = true)
+        private String groupId;
```
Comment (haifxu) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/UpdateCommand.java:49]: Please add description.

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/pojo/GroupInfo.java [hunk 3]
```diff
@@ -34,6 +35,7 @@ public class GroupInfo {
 
     @ParseStatus
     private String status;
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     private Date modifyTime;
```
Comment (haifxu) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/pojo/GroupInfo.java:39]: Other pojos with `Date` fields may also need this annotation.

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    public LogCommand() {
+        super("log");
+        jcommander.addCommand("group", new CreateGroup());
+    }
+
+    @Parameters(commandDescription = "Log group")
+    private static class CreateGroup extends AbstractCommandRunner {
+
+        @Parameter()
+        private java.util.List<String> params;
```
Comment (haifxu) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java:49]: Use `List<String>` pls.

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/UpdateCommand.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+import org.apache.inlong.manager.client.cli.util.ClientUtils;
+import org.apache.inlong.manager.pojo.sort.BaseSortConf;
+
+import java.io.File;
+
+@Parameters(commandDescription = "Update resource by json file")
+public class UpdateCommand extends AbstractCommand {
+
+    @Parameter()
+    private java.util.List<String> params;
```
Comment (haifxu) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/UpdateCommand.java:34]: Ditto

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java [hunk 6]
```diff
@@ -51,26 +51,38 @@ private static class CreateGroup extends AbstractCommandRunner {
 
         @Parameter(names = {"-f", "--file"},
                 converter = FileConverter.class,
-                required = true,
                 description = "json file")
         private File file;
 
+        //TODO:add gdbc connector and others
```
Comment (woofyzhao) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java:57]: Suggest leave a blank between start of comment // and _text_ which is the case in all other inlong source files.

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/DeleteCommand.java [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import org.apache.inlong.manager.client.api.InlongClient;
+import org.apache.inlong.manager.client.api.InlongGroup;
+import org.apache.inlong.manager.client.cli.util.ClientUtils;
+
+import java.util.List;
+
+@Parameters(commandDescription = "Delete resource by json file")
+public class DeleteCommand extends AbstractCommand {
```
Comment (woofyzhao) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/DeleteCommand.java:29]: Please add comment doc for public class

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java [hunk 8]
```diff
@@ -51,26 +51,38 @@ private static class CreateGroup extends AbstractCommandRunner {
 
         @Parameter(names = {"-f", "--file"},
                 converter = FileConverter.class,
-                required = true,
                 description = "json file")
         private File file;
 
+        // TODO:add gdbc connector and others
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java:57]: jdbc?

File: inlong-manager/manager-client-tools/src/test/resources/create_group.json [hunk 9]
```diff
@@ -41,21 +37,23 @@
...
-    "allMigration": false,
-    "databaseWhiteList": "etl",
-    "tableWhiteList": "person",
     "inlongGroupId": "test_group",
-    "sourceType": "BINLOG",
-    "specificOffsetFile": "mysql-bin.000001",
-    "specificOffsetPos": 6972,
-    "snapshotMode": "schema_only_recovery"
+    "sourceType": "FILE",
+    "agentIp": "123.456.789",
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/test/resources/create_group.json:42]: Please change to `127.0.0.1`.

File: inlong-manager/manager-client-tools/src/test/resources/test_config.json [hunk 10]
```diff
@@ -0,0 +1,9 @@
+{
+  "FlinkSortConf": {
+    "sortType": "flink",
+    "authentication": "NONE",
+    "serviceUrl": "123.456.789",
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/test/resources/test_config.json:5]: Please change to `127.0.0.1`, and add the default port.

File: inlong-manager/manager-client-tools/src/test/java/org/apache/inlong/manager/client/cli/TestCommand.java [hunk 11]
```diff
@@ -51,4 +51,23 @@ public void testCreateGroup() {
         String[] arg = {"create", "group", "-f", "src/test/resources/create_group.json"};
         Assertions.assertTrue(inlongAdminTool.run(arg));
     }
+
+    // @Test
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/test/java/org/apache/inlong/manager/client/cli/TestCommand.java:55]: Why comment out those test methods? Thanks.

File: inlong-manager/manager-client-tools/src/test/resources/create_group.json [hunk 12]
```diff
@@ -41,21 +37,23 @@
...
+    "dataNodeName": "hi2",
+    "serializationType": "csv",
+    "snapshot": "hi3",
+    "version": "4",
+    "status": "5",
+    "previousStatus": "6",
+    "creator": "me",
+    "modifier": "none",
+    "createTime": "2000-07-08 12:05:12",
+    "modifyTime": "2000-07-08 12:05:12",
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/test/resources/create_group.json:55]: Those fields are not required when creating a group.

**Review by haifxu — APPROVED** [review]

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+import org.apache.inlong.manager.client.cli.util.ClientUtils;
+import org.apache.inlong.manager.client.cli.util.PrintUtils;
+import org.apache.inlong.manager.pojo.group.InlongGroupBriefInfo;
+import org.apache.inlong.manager.pojo.group.InlongGroupPageRequest;
+
+import java.util.List;
+
+/**
+ * The log command used to get filter certain kinds of inlong groups
+ * Please refer to the document for parameters
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java:34]: Where is the link to the document?
Suggested remove this line.

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+        private String input;
+
+        @Override
+        void run() {
+            final int MAX_LOG_SIZE = 100;
+            try {
+                // for now only filter by one condition. TODO:support OR and AND, make a condition filter.
+                // sample input: inlongGroupId:test_group
+                if(input == null) {
+                    System.err.println("empty input");
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java:63]: "the input cannot be empty, for example: inlongGroupId:test_group"

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+                    return;
+                }
+                String[] inputs = input.split(":");
+                ClientUtils.initClientFactory();
+                InlongGroupClient groupClient = ClientUtils.clientFactory.getGroupClient();
+                InlongGroupPageRequest pageRequest = new InlongGroupPageRequest();
+                if (inputs.length<2 || StringUtils.isBlank(inputs[1])) {
+                    System.err.println("invalid input");
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/LogCommand.java:71]: "the input must contain ':'"

**Review by healchow — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**